src/nim_status/lib/shim.nim

```diff
@@ -4,3 +4,7 @@ import ../lib
 proc hashMessage*(message: string): string =
   let hash = lib.hashMessage(message)
   fmt("{{\"result\":\"{hash}\"}}")
+
+export generateAlias
+
+export saveAccountAndLogin
```

Comment:
See comment above re: `saveAccountAndLogin`. Even though this isn't the top-level, it seems best to not export a lib-shim until it's ready to serve as a stand-in for the status-go functionality.